src/_data/catalog/overrides.yml

```
@@ -60,3 +60,50 @@ items:
       accepts data in cloud-mode from web and mobile sources, and can accept data in device-mode from Analytics.js sources.
     cmode_type: mixed
     case: '10'
+- slug: friendbuy
+  display_name: Friendbuy
+  previous_names:
+  - FriendBuy
+  components:
+  - code: https://github.com/segment-integrations/analytics.js-integration-friendbuy
+    type: browser
+  connection_modes:
+    device:
+      web: true
+      mobile: false
+      server: false
+    cloud:
+      web: false
+      mobile: false
+      server: false
+    summary: accepts device-mode data only from Analytics.js.
+    cmode_type: device-only
+    case: '2'
+  settings:
+  - name: siteId
+    type: string
+    defaultValue: ''
+    description: >-
+      This is your **Site ID**. It is used to identify your account in our
+      platform so we can properly attribute referral data. You can find your
+      Site ID in the in Friendbuy web application at **Settings > Integration
+      Code**
+    required: true
+    label: Site ID
+  - name: siteWideWidgets
+    type: mixed
+    defaultValue: []
+    description: >-
+      By default, Friendbuy recommends you add a site wide overlay widget. You
```

```suggestion
      By default, Friendbuy recommends that you add a site wide overlay widget. You
```
This is minor, but I'd explicitly write out the relative pronoun here.  It helps with users who put our pages through online translators.